src/components/Pages/MainPage/MainPage.tsx

```diff
@@ -1,33 +1,41 @@
-import { useState } from 'react'
-import { Button } from 'decentraland-ui/dist/components/Button/Button'
-import reactLogo from '../../../assets/react.svg'
+import { useCallback, useState } from 'react'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Tabs } from 'decentraland-ui/dist/components/Tabs/Tabs'
+import { Divider } from 'decentraland-ui'
```

Comment:
Would you mind importing the Divider directly from the distributable?
```suggestion
import Divider from 'semantic-ui-react/dist/commonjs/elements/Divider/Divider'
```